go.mod

```diff
@@ -6,6 +6,7 @@ replace (
 	github.com/cometbft/cometbft => github.com/berachain/cometbft v1.0.1-0.20250424093341-6fe80335f54a
 	github.com/cometbft/cometbft/api => github.com/berachain/cometbft/api v1.0.1-0.20250424093341-6fe80335f54a
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.52.0-rc.1
+	github.com/ethereum/go-ethereum => ../bera-geth
```

Comment:
Noice

Reply:
Also do we have a public repo to depend on?

Reply:
So because of [these checks](https://github.com/berachain/beacon-kit/blob/public-key-in-FCU/state-transition/core/state_processor_payload.go#L182-L183), it now becomes a strict requirement to use bera-geth (since we're basically enforcing the EL level protocol rules in the CL). 

bera-geth is public (https://github.com/berachain/bera-geth/pull/21), and as soon as this branch is well tested, will merge it and release a tag to be used here